screenshots/generate_new_configs.py

```diff
@@ -0,0 +1,94 @@
+import os
+import io
+import pandas as pd
+import requests
+import yaml
+
+def load_urls_from_live_info(url):
+
+    content = requests.get(url).content
+    state_info_df = pd.read_csv(io.StringIO(content.decode('utf-8')), keep_default_na=False)
```

Comment:
just curious why the `keep_default_na=False`? (otherwise this looks pretty similar to the way links are read in backup_to_s3.py which makes sense)

Reply:
Pandas was converting the empty cell into NaN, which was just slightly weird to me... I thought empty cells should be empty strings, and this did that.